File: python/DPR.py
# deklarasi class
class Mahasiswa:

    # deklarasi private atribut
    __id = ""
    __name = ""
    __bidang = ""
    __partai = ""

    # constructor dengan parameter
    def __init__(self, id="", name="", bidang="", partai=""):
        self.__id = id
        self.__name = name
        self.__bidang = bidang
        self.__partai = partai

    # setter dan getter untuk masing masing attribut
    def get_id(self):
        return self.__id

    def set_id(self, id):
        self.__id = id

    def get_name(self):
        return self.__name

    def set_name(self, name):
        self.__name = name

    def get_bidang(self):
        return self.__bidang

    def set_bidang(self, bidang):
        self.__bidang = bidang

    def get_partai(self):
        return self.__partai

    def set_partai(self, partai):
        self.__partai = partai

File: php/DPR.php
<?php

// deklarasi class
class DPR
{
    // attibut private
    private $id;
    private $name;
    private $bidang;
    private $partai;

    public function __construct($od = '', $name = '', $bidang = '', $partai = '')
    {
        $this->id = $id;
        $this->name = $name;
        $this->bidang = $bidang;
        $this->partai = $partai;
    }

    // setter dan getter untuk setiap atribut
    public function get_id()
    {
        return $this->id;
    }
    public function set_id($id)
    {
        $this->id = $id;
    }

    public function get_name()
    {
        return $this->name;
    }
    public function set_name($name)
    {
        $this->name = $name;
    }

    public function get_bidang()
    {
        return $this->bidang;
    }
    public function set_bidang($bidang)
    {
        $this->bidang = $bidang;
    }

    public function get_partai()
    {
        return $this->partai;
    }
    public function set_partai($partai)
    {
        $this->partai = $partai;
    }
};
